Documentation/bash-scripts.md

```diff
@@ -0,0 +1,52 @@
+# Bash scripts
+
+A common way to use acbuild is via a bash script. There's a common template
+that the examples for acbuild follow, and it makes cleanup a little easier. The
+template is at the end of this file, and here we explain the reasoning behind
+the parts of it.
+
+The 1st line, `#!/usr/bin/env bash`, is simply good form to make sure the
+script is executed with bash as opposed to the current shell.
+
+The 2nd line, `set -e` makes the script exit in the event of an error. If some
+command doesn't exit correctly, due to something like a typo in a command, or
+the command in `acbuild run` encountering an error, it is better to end the
+script, instead of continuing on and producing an ACI with unintended values or
+contents.
+
+The 4th through the 7th lines check if the script is being run as the root
```

Comment:
If we're `-e` bailing on any nonzero return, why can't we cut this paragraph and these lines from the script? If I'm not root, script will fail and bail if it turns out I need to be root for some step, right? In other words, can we just leave the user handling to the environment. Make any sense?